integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/mutation/CarbonProjectForUpdateCommand.scala

```diff
@@ -169,6 +172,36 @@ private[sql] case class CarbonProjectForUpdateCommand(
           DeleteExecution
             .clearDistributedSegmentCache(carbonTable, segmentsToBeDeleted)(sparkSession)
 
+          // trigger prepriming
+
+          val prop = CarbonProperties.getInstance
+            .getProperty(CarbonCommonConstants.CARBON_ENABLE_INDEX_SERVER)
```

Comment:
This code is used from many places..Can we move it to some common method?

Reply:
done